resources/assets/components/HistoryModal/index.js

```diff
@@ -35,8 +64,11 @@ class HistoryModal extends React.Component {
             </div>
           </div>
 
-          <h3>Reportback History</h3>
-          <p>table of all the history goes here 📖</p>
+          <h3>📖 History 📖</h3>
+          <p>Below shows the 20 most recent changes to the member's signup. This includes changes to the quantity or why. If you need changes beyond the 20 listed here, please reach out to Team Bleed!</p>
+          <div className="container">
```

Comment:
yikes! What does this red mean?

Reply:
Ahhh yeah, this is just Github being a mess. Their syntax highlighting breaks whenever it encounters a `'` or some other random things in JSX blocks.